cmd/supercommand.go

```diff
@@ -86,3 +87,10 @@ func Info(i *cmd.Info) *cmd.Info {
 	info.ShowSuperFlags = []string{"show-log", "debug", "logging-config", "verbose", "quiet", "h", "help"}
 	return &info
 }
+
+// IsPiped determines if the command was used in a pipe and,
+// hence, it's stdin is not usable for user input.
+func IsPiped(ctx *cmd.Context) bool {
```

Comment:
Do we know if this could produce false positives?

Reply:
I have done the testing on several commands and scenarios and have not encountered any discrepancies... So, if there is such a possibility, we have not come across it yet.

This code is mirror image of what add-k8s command uses to get piped config, and that code is exercised excessively. So if it was unstable, it'd be discovered quickly.